packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -118,12 +119,14 @@ const getTags = (
       PostTag: PostTypeToTagV1[postType] ?? ArticlePostCardV1,
       AdTag: AdCardV1,
       PlaceholderTag: PlaceholderCardV1,
+      MarketingCTATag: MarketingCTACard,
     };
   }
   return {
     PostTag: isList ? PostList : PostTypeToTag[postType] ?? ArticlePostCard,
     AdTag: isList ? AdList : AdCard,
     PlaceholderTag: isList ? PlaceholderList : PlaceholderCard,
+    MarketingCTATag: isList ? MarketingCTAList : MarketingCTACard,
```

Comment:
Decided to split here to make it easier to maintain for future potentially different layouts :)